js/main.js

```diff
@@ -10,6 +10,8 @@ Site = {
       _this.onResize();
     });
 
+    _this.thinkOf();
```

Comment:
I think this should have been another feature. It isn't related to style everything